lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_action.rb

```diff
@@ -209,24 +214,36 @@ def self.is_firebasecmd_supported?(cmd)
         true
       end
 
+      def self.post_notes(app_id, release_id, release_notes)
+        payload = { releaseNotes: { releaseNotes: release_notes } }
+        if release_notes.nil? || release_notes.empty?
+          UI.message("No release notes passed in. Skipping this step.")
+          return
+        end
+        connection.post("#{v1_apps_path(app_id)}/releases/#{release_id}/notes", payload.to_json) do |request|
+          request.headers["Authorization"] = "Bearer " + auth_token
+        end
+        UI.message("Release notes have been posted.")
+      end
+
       def self.get_upload_token(app_id, binary_path)
         begin
           binary_hash = Digest::SHA256.hexdigest(File.open(binary_path).read)
         rescue Errno::ENOENT
-          UI.crash!(ErrorMessage::APK_NOT_FOUND)
+          UI.crash!(ErrorMessage::APK_NOT_FOUND + binary_path)
```

Comment:
I don't think this will be formatted well. Do you want a `: ` in between the two strings? Maybe a format string would be good: `"#{ErrorMessage::APK_NOT_FOUND}: #{binary_path}"`

There are a couple of instances of this in this PR, so if you fix this one please fix them all.